core/pom.xml

```diff
@@ -40,6 +40,11 @@
       <classifier>${avro.mapred.classifier}</classifier>
     </dependency>
     <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk</artifactId>
+      <version>${aws.java.sdk.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
@pwendell @yhuai here is what dependencies look like now-
```
[INFO] +- com.amazonaws:aws-java-sdk-s3:jar:1.9.16:compile
[INFO] |  \- com.amazonaws:aws-java-sdk-kms:jar:1.9.16:compile
[INFO] +- com.amazonaws:aws-java-sdk-sts:jar:1.9.16:compile
[INFO] |  \- com.amazonaws:aws-java-sdk-core:jar:1.9.16:compile
[INFO] |     +- commons-logging:commons-logging:jar:1.1.3:compile
[INFO] |     +- org.apache.httpcomponents:httpclient:jar:4.3.2:compile
[INFO] |     |  \- org.apache.httpcomponents:httpcore:jar:4.3.2:compile
[INFO] |     \- joda-time:joda-time:jar:2.5:compile
```
I could further exclude transitive deps if that helps.

Btw, there is a [known issue](https://github.com/aws/aws-sdk-java/issues/444) in aws sdk when using it with Java 8 u60 and joda-time < 2.8.1. So I had to update joda-time version to 2.8.1+ in my build at Netflix.